kustomization.yaml

```diff
@@ -1,2 +1,8 @@
-bases:
-- kustomize/base
\ No newline at end of file
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- kustomize/base
+# only enable this if the k8s cluster is 1.15+ and has resource quota support
```

Comment:
neverming - see you have added here. Though should it get reflected in the doc somewhere?

Reply:
It is in the doc, I just moved it to the operator deployment along with other kustomize commands